test/parallel/test-child-process-exec-stdout-stderr-data-string.js

```diff
@@ -4,20 +4,9 @@ const common = require('../common');
 const assert = require('assert');
 const exec = require('child_process').exec;
 
-var stdoutCalls = 0;
-var stderrCalls = 0;
-
 const command = common.isWindows ? 'dir' : 'ls';
-exec(command).stdout.on('data', (data) => {
-  stdoutCalls += 1;
-});
+exec(command).stdout.on('data', common.mustCall((data) => {}));
 
-exec('fhqwhgads').stderr.on('data', (data) => {
+exec('fhqwhgads').stderr.on('data', common.mustCall((data) => {
   assert.strictEqual(typeof data, 'string');
-  stderrCalls += 1;
-});
-
-process.on('exit', () => {
-  assert(stdoutCalls > 0);
-  assert(stderrCalls > 0);
-});
+}));
```

Comment:
Yes. It was in the CI run actually. I'm going to fix it up when I push some changes.